Lib/axisregistry/data/element_shape.textproto

```diff
@@ -0,0 +1,9 @@
+#Element Shape, based on https://github.com/rosettatype/handjet
+tag: "ESHP"
```

Comment:
Decision made to revise this to "ELSP" from "ESHP"

Reply:
Addressed in https://github.com/googlefonts/axisregistry/pull/56/commits/aa483aa61e89b1f79079119cf239f6c4dc3a7047